homeworks/12-mars-mobx/src/App/App.styled.tsx

```diff
@@ -0,0 +1,43 @@
+import styled from '@emotion/styled';
+
+export const Navigation = styled.ul`
+  display: flex;
+  justify-content: center;
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+  padding-top: 20px;
```

Comment:
For better responsive design it is advisable to use `rem` and `em` units for margins, padding.
However for borders it is fine to use pixels because half pixel lines don't look good.